packages/snap/package.json

```diff
@@ -42,17 +51,17 @@
     "@typescript-eslint/eslint-plugin": "5.42.1",
     "@typescript-eslint/parser": "5.42.1",
     "dotenv": "16.4.5",
-    "eslint": "8.45.0",
-    "eslint-config-prettier": "8.5.0",
-    "eslint-plugin-import": "2.26.0",
-    "eslint-plugin-jest": "27.1.5",
-    "eslint-plugin-jsdoc": "41.1.2",
-    "eslint-plugin-n": "15.7.0",
-    "eslint-plugin-prettier": "4.2.1",
-    "eslint-plugin-promise": "6.1.1",
+    "eslint": "8.57.0",
+    "eslint-config-prettier": "9.1.0",
+    "eslint-plugin-import": "2.29.1",
+    "eslint-plugin-jest": "27.9.0",
+    "eslint-plugin-jsdoc": "48.8.0",
+    "eslint-plugin-n": "17.9.0",
+    "eslint-plugin-prettier": "5.2.1",
+    "eslint-plugin-promise": "6.4.0",
     "jest": "29.5.0",
-    "prettier": "2.7.1",
-    "prettier-plugin-packagejson": "2.2.11",
+    "prettier": "3.3.3",
```

Comment:
`@metamask/auto-changelog` doesn't work with Prettier 3 unfortunately.

Reply:
ok, I'll revert to an earlier version